tech.feliciano.pocket-casts.yml

```diff
@@ -0,0 +1,60 @@
+app-id: tech.feliciano.pocket-casts
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
```

Comment:
```suggestion
  - --socket=x11
```

Reply:
No Wayland support there?

Reply:
Not in the chromium-verse yet (without flags twiddling and special compiles). Should be in mainline chromium soon, thence to electron eventually.